ios/brave-ios/Sources/DataImporter/Images.xcassets/password_conflict_logo.imageset/Contents.json

```diff
@@ -0,0 +1,22 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "filename" : "Group 2@2x.png",
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "filename" : "Group 2@3x.png",
+      "idiom" : "universal",
+      "scale" : "3x"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
+}
```

Comment:
this icon looks the same as `failure_import_logo` above

Reply:
Ahh yeah. They're the same in the design. I guess I could delete this one.